### ISSUE TITLE
bug: App Not Drawing Behind Navigation Bar on Patching Screen

### ISSUE THREAD

**Issue by Dan-Banfield** [ReVanced/revanced-manager#1783]
### Bug description

On the patching screen, it seems the app is not in edge-to-edge mode because it does not draw behind the system navigation bar.

![1000020943](https://github.com/ReVanced/revanced-manager/assets/73112854/8c41a347-1b1b-413a-bc89-82e75b0e9462)

### Version of ReVanced Manager and version & name of app you are patching

Version 1.19.3
App being patched: any

### Installation method

Regular

### ReVanced Manager logs

```shell
Not relevant.
```

### Patch logs

```shell
Not relevant.
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager

**Comment by snehithssv** [comment]
So, while on patching screen, the bottom nav-bar should be hidden?? @Dan-Banfield

**Comment by Dan-Banfield** [comment]
> So, while on patching screen, the bottom nav-bar should be hidden?? @Dan-Banfield

No, not hidden, just the app should draw behind it in-line for the upcoming mandate in Android 15.

**Comment by snehithssv** [comment]
So, app is to be forced to be in edgeToEdge instead of leanBack/immersive.?
More like this...??
![Screenshot_1718089769](https://github.com/ReVanced/revanced-manager/assets/112557710/c173bf34-1a32-4999-95d9-17bc96a60539)

 And if yes, should it be this way only in patch screen on for the whole application??

**Comment by Dan-Banfield** [comment]
In the screenshot you provided, the navigation bar at the bottom is still black. Instead, it should be drawn behind. Please see: https://developer.android.com/about/versions/15/behavior-changes-15#:~:text=Edge-to-edge%20enforcement,because%20most%20components%20apply%20insets.

**Comment by snehithssv** [comment]
Thanks a lot for helping me out... @Dan-Banfield 
[projc.webm](https://github.com/ReVanced/revanced-manager/assets/112557710/86d118d1-4fbe-40db-8920-1df1e7399665)
So, is this the desired functionality?

**Comment by Dan-Banfield** [comment]
> Thanks a lot for helping me out... @Dan-Banfield 
> [projc.webm](https://github.com/ReVanced/revanced-manager/assets/112557710/86d118d1-4fbe-40db-8920-1df1e7399665)
> So, is this the desired functionality?

Yes, that looks great! Nice job.

**Comment by ILoveOpenSourceApplications** [comment]
This can be closed right?

**Comment by validcube** [comment]
> This can be closed right?

Nice catch, thanks.
